resources/public/api/conf/1.0/examples/businesses/create_request.json

```
@@ -0,0 +1,23 @@
+{
+  "typeOfBusiness":"uk-property",
+  "tradingName":"Business McBusiness",
+  "firstAccountingPeriodStartDate": "2019-09-30",
+  "firstAccountingPeriodEndDate":"2020-02-29",
+  "latencyDetails": {
+    "latencyEndDate": "2020-02-27",
+    "taxYear1": "2019",
+    "latencyIndicator1": "Annual",
+    "taxYear2": "2020",
+    "latencyIndicator2": "Annual"
+  },
```

Are you sure these tax years are correct? The tech spec says they should be in the `YYYY-YY` format.

Also, from what I've seen in other backend endpoints, the YYYY tax year format isn't a string, but a number. Where did you see "YYYY"?
```suggestion
  "latencyDetails": {
    "latencyEndDate": "2020-02-27",
    "taxYear1": "2018-19",
    "latencyIndicator1": "Annual",
    "taxYear2": "2019-20",
    "latencyIndicator2": "Annual"
  },
```